src/values.js

```diff
@@ -123,6 +123,7 @@ wrapToString(Ref, function() {
     indexes: 'Index',
     functions: 'Function',
     roles: 'Role',
+    accessProviders: 'AccessProvider',
```

Comment:
`access_providers: 'AccessProvider'`

Reply:
@marrony That's what I had originally, however, when snake-casing it, the pretty print results in `Access_provider` and fails our tests.

Reply:
there is no sneak case conversion in this place, this is used to convert from schema name to the function name

Reply:
I had all three instances as `access_provider` in `value.js` before changing them to `accessProviders`. Still got that result 🤔

Reply:
two different things going on here:

1. Rendering of `AccessProvider("blah")`
  https://github.com/fauna/faunadb-js/blob/master/src/values.js#L134
  to fix this you just need to add `access_providers: 'AccessProvider'` on `constructors` object

2. Rendering of `AccessProviders([db])`
  https://github.com/fauna/faunadb-js/blob/master/src/values.js#L131
  this would involve more change, but for the sake of making the tests pass you can just add 
  `      if (ref.id === 'access_providers') return 'AccessProviders(' + db + ')'` before the line 131

Reply:
@marrony Thanks for the notes. I was just doing some investigation and was about to ping you about item 2 since I traced the issue there as well 👍

Reply:
for the item 2 you can do some fancy/generic stuff like here https://github.com/fauna/faunadb-js/blob/master/src/Expr.js#L150 or a simple `if` should be enough